### DISCUSSION TITLE
CRUD operations not reflecting immediately on Next.js app deployed with Vercel

### DISCUSSION THREAD

**Discussion by samyakpiya** [vercel/next.js#51535] [Help]
### Summary

Hello, I'm a budding full-stack developer and have recently built a CRUD web application using Next.js, following a tutorial. The app is deployed on Vercel.

During local development, I've noticed that when I create, update, or delete a post, the changes reflect immediately. However, when working with the online deployment through Vercel, the changes aren't showing up live. The data remains the same as before any updates were made. It's only after manually redeploying the application that the changes become visible.

Based on my understanding, I suspect this might be due to Vercel's default caching strategy ("Serverless Pre-Rendering" or SPR). This would explain why changes reflect after redeployment (since cache invalidation is a part of the deployment process).

Here's the link to the live website: https://promptbookapp.vercel.app/
And the link to the GitHub repository: https://github.com/samyakpiya/promptbook

Would anyone be able to shed some light on this issue? Specifically, I'm looking for suggestions on how I can get changes to reflect immediately on the live site without needing to manually redeploy the application each time.

I appreciate any help or insights you can provide. Thanks in advance!

**Comment by CrissyPH** [comment]
did you find the answer? im having the same error
